test/LayoutTest.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import test from 'tape';
+
+import Layout from '../src/Layout';
+
+test('<Layout>', t => {
+  const wrapper = shallow(<Layout skin="blue">
+      <p>asdf</p>
+    </Layout>);
+
+  t.ok(wrapper.hasClass('sidebar-mini'), 'has (default) sidebar-mini class');
+  t.ok(wrapper.children().hasClass('wrapper'), 'renders child wrapper');
```

Comment:
Checking for the wrapper class is good, but we also need to check that the children are rendered inside of the wrapper.